### PR TITLE
SPCTR-1665 Deprecated block and post child block are visible on repository. Showing total 40 blocks.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: scripts/generate-blocks-placeholder.js [hunk 1]
```diff
@@ -30,7 +30,10 @@ const registerBlocks = glob
 		const blockSlug = getBlockSlug( content );
 		const blockTitle = getBlockTitle( content );
 
-		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || './src/blocks/column/block.js' === file ) {
+		const excludedBlock = ['./src/blocks/post/post-button/block.js', './src/blocks/post/post-excerpt/block.js', './src/blocks/post/post-image/block.js', './src/blocks/post/post-masonry/block.js', './src/blocks/post/post-meta/block.js',
+		'./src/blocks/post/post-taxonomy/block.js', './src/blocks/post/post-title/block.js', './src/blocks/wp-search/block.js','./src/blocks/gf-designer/block.js', './src/blocks/cf7-designer/block.js', './src/blocks/columns/block.js', './src/blocks/section/block.js'];
+
+		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || './src/blocks/column/block.js' === file || excludedBlock.includes(file) ) {
```
Comment (sandeshjangam) [inline scripts/generate-blocks-placeholder.js:36]:
```suggestion
		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || './src/blocks/column/block.js' === file || excludedBlock.includes( file ) ) {
```

**Review by RajkiranBagal — APPROVED** [review]

File: scripts/generate-blocks-placeholder.js (hunk 1 above)
Comment (sandeshjangam) [inline scripts/generate-blocks-placeholder.js:36]: We can add this in excluded list also
`'./src/blocks/column/block.js' === file`

File: scripts/generate-blocks-placeholder.js [hunk 2]
```diff
@@ -30,7 +30,10 @@ const registerBlocks = glob
 		const blockSlug = getBlockSlug( content );
 		const blockTitle = getBlockTitle( content );
 
-		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || './src/blocks/column/block.js' === file ) {
+		const excludedBlock = ['./src/blocks/post/post-button/block.js', './src/blocks/post/post-excerpt/block.js', './src/blocks/post/post-image/block.js', './src/blocks/post/post-masonry/block.js', './src/blocks/post/post-meta/block.js',
```
Comment (sandeshjangam) [inline scripts/generate-blocks-placeholder.js:33]:
```suggestion
		const excludedBlocks = ['./src/blocks/post/post-button/block.js', './src/blocks/post/post-excerpt/block.js', './src/blocks/post/post-image/block.js', './src/blocks/post/post-masonry/block.js', './src/blocks/post/post-meta/block.js',
```

File: scripts/generate-blocks-placeholder.js (hunk 1 above)
Comment (AbhijitNage123) [reply scripts/generate-blocks-placeholder.js]: Done

File: scripts/generate-blocks-placeholder.js (hunk 2 above)
Comment (AbhijitNage123) [reply scripts/generate-blocks-placeholder.js]: Done

File: scripts/generate-blocks-placeholder.js [hunk 3]
```diff
@@ -30,7 +30,10 @@ const registerBlocks = glob
 		const blockSlug = getBlockSlug( content );
 		const blockTitle = getBlockTitle( content );
 
-		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || './src/blocks/column/block.js' === file ) {
+		const excludedBlocks = ['./src/blocks/post/post-button/block.js', './src/blocks/post/post-excerpt/block.js', './src/blocks/post/post-image/block.js', './src/blocks/post/post-masonry/block.js', './src/blocks/post/post-meta/block.js',
+		'./src/blocks/post/post-taxonomy/block.js', './src/blocks/post/post-title/block.js', './src/blocks/wp-search/block.js','./src/blocks/gf-designer/block.js', './src/blocks/cf7-designer/block.js', './src/blocks/columns/block.js', './src/blocks/column/block.js', './src/blocks/section/block.js'];
+
+		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' )|| excludedBlocks.includes( file ) ) {
```
Comment (sandeshjangam) [inline scripts/generate-blocks-placeholder.js:36]:
```suggestion
		if ( ! blockSlug || ! blockTitle || -1 !== file.indexOf( 'child' ) || excludedBlocks.includes( file ) ) {
```

Comment (AbhijitNage123) [reply scripts/generate-blocks-placeholder.js]: Done

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**